sandbox/web/src/main/java/org/kaaproject/kaa/sandbox/web/client/i18n/SandboxConstants.java

```diff
@@ -2,7 +2,7 @@
 
 /**
  * Interface to represent the constants contained in resource bundle:
- * 	'/home/dyosick/projects/github_kaa/kaa/sandbox/web/src/main/java/org/kaaproject/kaa/sandbox/web/client/i18n/SandboxConstants.properties'.
+ * 'kaa/sandbox/web/src/main/java/org/kaaproject/kaa/sandbox/web/client/i18n/SandboxConstants.properties'.
```

Comment:
Maybe, I have opened separate ticket for this http://jira.kaaproject.org/browse/KAA-418